packages/plugin-web-attribution-browser/src/web-attribution.ts

```diff
@@ -26,7 +27,9 @@ export const webAttributionPlugin: CreateWebAttributionPlugin = function (option
         storage.get(storageKey),
       ]);
 
-      if (isNewCampaign(currentCampaign, previousCampaign, pluginConfig)) {
+      const _isNewSession = isNewSession(config.sessionTimeout, config.lastEventTime);
```

Comment:
nit: `_isNewSession` bothers me a little since it looks identical to the function at first glance. additionally, there are existing conventions that use the practice of pretending variable names with underscore.

1. Intentionally unused variable ([ref](https://www.sonarsource.com/blog/common-typescript-issues-no-3-unused-local-variables-and-functions/#:~:text=Naming%20a%20variable%20that%20is%20unused%20with%20an%20underscore%2C%20or%20prefixing%20a%20variable%20name%20with%20an%20underscore%2C%20are%20conventions%20to%20show%20other%20developers%20that%20we%20had%20to%20name%20the%20variable%2C%20but%20we%20don%27t%20intend%20to%20use%20it.))
2. Private variables ([ref](https://dev.to/somedood/emulating-private-variables-in-javascript-with-closures-and-factory-functions-2314#:~:text=It%20is%20simply%20the%20convention%20of%20prepending%20an%20underscore%20(_)%20to%20a%20variable%20name.%20This%20is%20done%20to%20indicate%20that%20a%20variable%20is%20private%20and%20should%20not%20be%20toyed%20with.))

Reply:
Any recommendation to the variable name?

Reply:
change it to `isEventInNewSession`.